### DISCUSSION TITLE
VOX ( UV-K5 EQZUMER V,0 22)

### DISCUSSION THREAD

**Discussion by UB7KOM** [egzumer/uvk5-chirp-driver#35] [Ideas]
Dear software developer for UV-K5, is it possible to increase the transmission delay time when VOX mode is on from 2 seconds to 4 seconds and post the finished firmware or send it by email.  mail?

**Comment by egzumer** [comment]
I have never adjusted it, I don't know where it is located. You can try this firmware: https://github.com/kamilsss655/uv-k5-firmware-custom
I believe there is a VOX delay adjustment if I'm not mistaken.

**Comment by UB7KOM** [comment]
I can’t find and download firmware 2.1.27.  The description says it should work.  give a link to download the firmware.
